ReSwiftDemo/Services/FlowManager.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import ReSwift
+import UIKit
+import SwinjectStoryboard
+
+class FlowManager: StoreSubscriber {
+	
+    private let store: Store<AppState>
+    private let appUserInteractions: AppUserInteractions
+    
+    init(store: Store<AppState>, appUserInteractions: AppUserInteractions) {
+        self.store = store
+        self.appUserInteractions = appUserInteractions
+        self.subscribe()
+    }
+    
+    func subscribe() {
+        self.store.subscribe(self) { subscription in
+            subscription.select { state in state.flowState }.skipRepeats()
+        }
+    }
+	
+    func newState(state: FlowState) {
+        print("Set Flow: \(state.flow)")
+        
+        if state.flow == .splashScreen {
+            self.appUserInteractions.launchApp()
+        } else if state.flow == .signIn {
+            self.showScreen(name: "LoginViewController")
```

Comment:
`"LoginViewController"` jest troche "magiczny" ;)

Reply:
Zgadza się, celowo już pominąłem niektóre szczegóły, ofc ostatecznie bym to wydzielił do jakiś stałych.